biogeochem/EDCanopyStructureMod.F90

```diff
@@ -468,7 +493,7 @@ subroutine DemoteFromLayer(currentSite,currentPatch,i_lyr)
                   currentSite%demotion_rate(currentCohort%size_class) = &
                        currentSite%demotion_rate(currentCohort%size_class) + currentCohort%n
                   currentSite%demotion_carbonflux = currentSite%demotion_carbonflux + &
-                       currentCohort%b_total() * currentCohort%n
+                       (leaf_c + store_c + fnrt_c + sapw_c + struct_c) * currentCohort%n
```

Comment:
@rgknox so here "struct_c" is structural carbon pool correct?  As in what we used to call "dead"?

Reply:
yup, exactly